Dockerfile

```diff
@@ -0,0 +1,34 @@
+## golang image created 2021-06-24T00:31:06.02014601Z 
+FROM docker.io/library/golang@sha256:be99fa59acd78bb22a41bbc1e15ebfab2262498ee0c2e28c3d09bc44d51d1774 AS builder
+WORKDIR /go/src/github.com/komish/preflight
+COPY . .
+RUN make build
+
+# ubi-minimal image created 2021-06-01T12:12:46.922866Z
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:0ccb9988abbc72d383258d58a7f519a10b637d472f28fbca6eb5fab79ba82a6b
+
+# Add preflight binary
+COPY --from=builder /go/src/github.com/komish/preflight/preflight /usr/local/bin/preflight
+
+# Install dependencies
+RUN microdnf install \
+    buildah \
+    bzip2 \
+    gzip \
+    iptables \
+    podman \
+    python3-pip \
+    skopeo
```

Comment:
we need `openscap-utils` as well for the vulnerability check

Reply:
Thanks, I will add it and update the PR.

Reply:
I added the openscap-utils dependency, thanks for catching that!

Reply:
It does not appear that the `openscap-utils` is available through microdnf in the ubi-minimal base image. However, I see it on my local Fedora 34 machine, so I am wondering if the repo containing that package is not available by default in the ubi-minimal image. I also do not see that package being installed in as part of the Vagratfile, unless it is being included as a dependency of something else that we have listed.

Reply:
Vagrantfile has been updated to install openscap: https://github.com/komish/preflight/pull/37/files